src/main/java/ratpack/zipkin/internal/MethodCapturingRequestSpec.java

```diff
@@ -60,7 +67,8 @@ public RequestSpec headers(Action<? super MutableHeaders> action) throws Excepti
 
   @Override
   public RequestSpec method(HttpMethod method) {
-    capturedMethod = method;
+    this.capturedMethod = method;
+    span.set(impl.handler.handleSend(impl.injector, getHeaders(), this));
```

Comment:
Ideally the client would have more callbacks for these things but I think this is the closest thing we have to an 'onRequestSent' callback.